style.css

```diff
@@ -0,0 +1,745 @@
+/* end form */
+
+main h2 {
+    margin: 100px 0 0;
+    text-align: center;
+}
+.table-container p {
+    text-align: center;
+    color: #d9d9d9;
+}
+/* input domain start */
+
+.input-domain-container {
+    display: flex;
+    position: relative;
+    width: 890px;
+    margin: auto;
+    border: 1px solid #d9d9d9;
+}
+.input-domain-container p {
+    margin: 6px 15px 0 18px;
+    font-size: 17px;
+    color: #d9d9d9;
+    
+}
+.input-domain-container::before {
+    content: "";
+    width: 1px;
+    height: 25px;
+    background-color: #d9d9d9;
+    position: relative;
+    top: 5px;
+    left: 67px;
+}
+.input-domain-container::after {
+    content: "";
+    width: 1px;
+    height: 25px;
+    background-color: #d9d9d9;
+    position: relative;
+    top: 5px;
+    right: 80px;
+}
+.input-domain {
+    height: 25px;
+    width: 80%;
+    margin: 5px 0;
+    border: none;
+    padding: 0;
+    outline: none;
+}
+
+.input-domain-select {
+    width: 80px;
+    border: none;
+    outline: none;
+    background-color: #fff;
+    font-size: 16px;
+    text-align: right;
+    color: #d9d9d9;
+}
+
+/* price table start */
+
+.price-container {
+    width: 890px;
+    border: 1px solid #d9d9d9;
+    margin: 25px auto;
+}
+.domain-table {
+    height: 130px;
+    width: 100%;
+    text-align: center;
+    border-spacing: 0px;
+}
+
+.domain-table td {
+    width: 25%;
+    padding: 0;
+    position: relative; 
+    color: #d9d9d9;
+}
+
+.domain-table td::after {
+    content: "";
+    position: absolute;
+    top: 15%;
+    left: 100%;
+    width: 1px;
+    height: 70%;
+    background-color: #d9d9d9;
+}
+.table-border td::before {
+    content: "";
+    position: absolute;
+    top: 100%;
+    left: 5%;
+    width: 91%;
+    height: 1px;
+    background-color: #d9d9d9;
+}
+
+/* price card start */
+.price-card-background {
+    height: 250px;
+    background-image: url(images/work1.jpg);
+    background-size: cover;
+    margin: 60px 0 350px;
+}
+.price-card-container {
+    display: flex;
+    width: 90%;
+    min-width: 1024px;
+    justify-content: space-between;
+    margin: auto;
+}
+.price-card {
+    width: 250px;
+    height: 480px;
+    margin: 50px 0;
+    background-color: #fff;
+    border-top: 5px solid #f9d423;
+    box-shadow: 0 2px 10px #d2d2d269;
+    transition: 500ms cubic-bezier(0, 0, 0.2, 1);
+}
+.price-card:hover {
+    transition: 500ms cubic-bezier(0, 0, 0.2, 1);
+    transform: scale(1.01 , 1.01);
+    box-shadow: 0px 0px 16px #1c1c1c;
+}
+.price-card h4 {
+    text-align: center;
+}
+.price-table {
+    width: 80%;
+    margin: auto;
+    font-size: 15px;
+    border-top: 1px solid #d9d9d9;
+    border-bottom: 1px solid #d9d9d9;
+}
+.price-table td:nth-last-child(1) {
+  text-align: right;
+  height: 30px;
+}
+.price-card table {
+    color: #e1e1e1;
+}
+
+.basic-package td:nth-last-child(1) {
+    color: #f9d423;
+}
+.normal-card td:nth-last-child(1) {
+    color: #7f8c8d;
+}
+.normal-card {
+    border-color: #7f8c8d;
+}
```

Comment:
why don't you merge last two rules '.normal-card td:nth-last-child(1)' and '.normal-card' ?